static/app/views/feedback/feedbackListPage.tsx

```diff
@@ -22,50 +24,50 @@ import FluidHeight from 'sentry/views/replays/detail/layout/fluidHeight';
 interface Props extends RouteComponentProps<{}, {}, {}> {}
 
 export default function FeedbackListPage({}: Props) {
-  const location = useLocation();
-  const query = useFeedbackListQueryParams({
-    location,
-    queryReferrer: 'feedback_list_page',
-  });
-
   const organization = useOrganization();
+  const location = useLocation<FeedbackItemLoaderQueryParams>();
 
+  const queryView = useFeedbackListQueryView({
+    queryReferrer: 'feedback_list_page',
+  });
   const feedbackSlug = decodeScalar(location.query.feedbackSlug);
 
   return (
-    <SentryDocumentTitle title={t(`Bug Reports`)} orgSlug={organization.slug}>
-      <FullViewport>
-        <Layout.Header>
-          <Layout.HeaderContent>
-            <Layout.Title>
-              {t('Bug Reports')}
-              <PageHeadingQuestionTooltip
-                title={t(
-                  'Feedback submitted by users who experienced an error while using your application, including their name, email address, and any additional comments.'
+    <FeedbackDataContext queryView={queryView} initialDate={new Date()}>
+      <SentryDocumentTitle title={t(`Bug Reports`)} orgSlug={organization.slug}>
+        <FullViewport>
+          <Layout.Header>
+            <Layout.HeaderContent>
+              <Layout.Title>
+                {t('Bug Reports')}
+                <PageHeadingQuestionTooltip
+                  title={t(
+                    'Feedback submitted by users who experienced an error while using your application, including their name, email address, and any additional comments.'
+                  )}
+                  docsUrl="https://docs.sentry.io/product/user-feedback/"
```

Comment:
should we remove this URL for now?

Reply:
`docsUrl` is required by the tooltip, so we'd have to remove the whole thing

lets take it out. better to not have it, and add it later, than to have it be wrong and no one notices